packages/shared/src/lib/core/ledger/actions/checkOrConnectLedgerAsync.ts

```diff
@@ -20,7 +20,7 @@ export function checkOrConnectLedgerAsync(
                 hideClose: true,
                 props: {
                     ledgerAppName,
-                    onSuccess: resolve,
+                    onContinue: resolve,
```

Comment:
lets make the prop the same for `ConnectLedgerPopup` and `UnlockStrongholdPopup`. 
I think `onSuccess` is the better of both